Makefile

```diff
@@ -214,7 +221,7 @@ appimage:
 appimage-%:
 	bash scripts/genappimage.sh $*
 
-lint: check-single-includes clint lualint _opt_pylint _opt_shlint _opt_commitlint
+lint: check-single-includes clint _opt_stylua lualint _opt_pylint _opt_shlint _opt_commitlint
```

Comment:
`lint` should only report errors. Do we need a separate `fmt` umbrella for tasks that autoformat stuff?

Reply:
`stylua --check` does linting only, it doesn't format.